recipes/setuptools/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "22.0.5" %}
+
+package:
+  name: setuptools
+  version: {{ version }}
+
+source:
+  fn: setuptools-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
+  md5: 869f3029dcc66a64ba39875e2a2f044a
+  patches:
+    # modify setuptools to barf if used in conda build (encourage people to add all deps in meta.yaml)
+    - nodownload.patch
+    # Don't install certifi or wincertstore
+    - setup.patch
+
+build:
+  number: 0
+  entry_points:
+    - easy_install = setuptools.command.easy_install:main
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  commands:
+    - easy_install --help
+  imports:
+    - setuptools
+    - easy_install
+    - pkg_resources
+
+about:
+  home: https://pypi.python.org/pypi/setuptools
+  license: PSF or ZPL
```

Comment:
So it says on PyPI that this MIT (via a [classifier]( https://pypi.python.org/pypi/setuptools/22.0.5#code-of-conduct )).